src/test/java/teammates/test/cases/ui/browsertests/InstructorCoursesPageUiTest.java

```diff
@@ -65,6 +67,13 @@ public static void classSetup() throws Exception {
         
         /* Explanation: Gets a browser instance to be used for this class. */
         browser = BrowserPool.getBrowser();
+        
+        /* 
+         * Any entity that is created in previous test run must be deleted.
+         * If that previous test run fails, the entity persists and that will
+         * break tests.
+         */
+        BackDoor.deleteCourse(validCourse.id); // delete if it exists
```

Comment:
If you're interested to try, just fail the test anytime _after line 217, before line 269_ is executed. I can guarantee that the subsequent runs will be a 100% failure at line 128.